cmd/extract_from_docker.go

```diff
@@ -272,6 +283,68 @@ func copyFromContainer(cli *dockerClient.Client, containerID string, containerPa
 	return nil
 }
 
+func copyWholeContainer(cli *dockerClient.Client, containerID string, hostBaseDir string) error {
+	err := os.MkdirAll(path.Dir(hostBaseDir), 0764)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fileReader, _, err := cli.CopyFromContainer(context.Background(), containerID, "/")
+	if err != nil {
+		return err
+	}
+	defer fileReader.Close()
+
+	tr := tar.NewReader(fileReader)
+
+	for {
+		header, err := tr.Next()
+
+		switch {
+
+		// if no more files are found return
+		case err == io.EOF:
+			return nil
+
+		// return any other error
+		case err != nil:
+			return err
+
+		// if the header is nil, just skip it (not sure how this happens)
+		case header == nil:
+			continue
+		}
+
+		target := filepath.Join(hostBaseDir, header.Name)
+		fmt.Println(target)
```

Comment:
i'd only show this if debug/log verbosity is turned on

Reply:
Gotcha, forgot to remove that

Reply:
removed it